react-frontend/src/sagas/loginSaga.js

```diff
@@ -0,0 +1,24 @@
+import { put, takeEvery, call } from "redux-saga/effects";
+
+import PostJson from "utils/postJson";
+
+export function* userLogin(action) {
+  try {
+    const response = yield call(PostJson, {
+      path: "/oauth/google",
```

Comment:
@onkar-josh Where are maintaining the Google app credentials required for Google OAuth? i.e client IDs, secret keys?

Reply:
@mayuriardad we will maintain those credentials in component where google button will be located and pass them as a environment variables.

Reply:
@onkar-josh Similar to how u have used ENV variables on node side, do not use ENV variables directly in code. Instead, assign all ENV variables to constants (we already have a constants file) and use those constants in ur code.